### DISCUSSION TITLE
Camelot Pro compatible

### DISCUSSION THREAD

**Discussion by mandragora73** [apohl79/audiogridder#1042] [Ideas]
Camelot Pro is a useful software for live music. In this moment Audiogridder in not compatiple with it.

**Comment by apohl79** [comment]
I did a quick test with the demo version of camelot and it loads AG just fine. So whats not working?

**Reply by mandragora73** [reply]
Thanks fou your answer. I use a Win 10 system. I've installed AG with no problems. But Camelot Pro "doesn't see" AG VST, instead of Studio One (for example) that "see" AG quietly.

**Reply by apohl79** [reply]
I did the same test on Windows 10 and AG shows up and loads successfully. I did not touch any of camelots default settings. Maybe you changed the VST location?

**Comment by mandragora73** [comment]
It was a simply problem of VST path location! It's solved. Thanks!
